inventory/targets/examples/artifactory.yml

```diff
@@ -0,0 +1,45 @@
+# Sources:
+# https://github.com/kapicorp/kapitan/blob/master/docs/compile.md#helm
+# https://artifacthub.io/packages/helm/jfrog/artifactory
+
+classes:
+  - common
+
+parameters:
+  target_name: artifactory
+
+  kapitan:
+    dependencies:
+    - type: https
+      source: https://charts.jfrog.io/artifactory/helm/artifactory-11.7.6.tgz
```

Comment:
is there no chart we can download with the "helm" type?

Reply:
sure there is.
I just used helm as described here: https://github.com/kapicorp/kapitan/blob/master/docs/compile.md#helm